src/k2/httpproxy/HTTPProxy.h

```diff
@@ -26,6 +26,18 @@ Copyright(c) 2022 Futurewei Cloud
 #include <k2/module/k23si/client/k23si_client.h>
 
 namespace k2 {
+namespace dto {
+// Request to get a schema
+struct GetSchemaRequest {
+    String collectionName;
+    String schemaName;
+    uint32_t schemaVersion = 0;
```

Comment:
This field should be a signed int64. From the client library, there is a special value we use to indicate "any version". You should place it here as a constant:
```
static constexpr int64_t ANY_VERSION = -1;
```

Reply:
Done